integration-tests/api/__tests__/admin/draft-order/draft-order.js

```diff
@@ -517,6 +514,98 @@ describe("/admin/draft-orders", () => {
       )
     })
 
+    it("creates a draft order with fixed discount amount allocated to the total", async () => {
+      const api = useApi()
+
+      const testVariantId = "test-variant"
+      const testVariant2Id = "test-variant-2"
+
+      const discount = await simpleDiscountFactory(dbConnection, {
+        code: "test-fixed",
+        regions: ["test-region"],
+        rule: {
+          type: "fixed",
+          allocation: "total",
+          value: 1000,
+        },
+      })
+
+      const payload = {
+        email: "oli@test.dk",
+        shipping_address: "oli-shipping",
+        discounts: [{ code: discount.code }],
+        items: [
+          {
+            variant_id: testVariantId,
+            quantity: 2,
+            metadata: {},
+          },
+          {
+            variant_id: testVariant2Id,
+            quantity: 2,
+            metadata: {},
+          },
+        ],
+        region_id: "test-region",
+        customer_id: "oli-test",
+        shipping_methods: [
+          {
+            option_id: "test-option",
+          },
+        ],
+      }
+
+      const response = await api.post(
+        "/admin/draft-orders",
+        payload,
+        adminReqConfig
+      )
+
+      expect(response.status).toEqual(200)
+
+      const draftOrder = response.data.draft_order
+      const lineItem1 = draftOrder.cart.items.find(
+        (item) => item.variant_id === testVariantId
+      )
+      const lineItem2 = draftOrder.cart.items.find(
+        (item) => item.variant_id === testVariant2Id
+      )
+
+      expect(draftOrder.cart.items).toHaveLength(2)
+
+      expect(lineItem1).toEqual(
+        expect.objectContaining({
+          variant_id: testVariantId,
+          unit_price: lineItem1.unit_price,
+          quantity: lineItem1.quantity,
```

Comment:
nit: you're testing the same object that you queried. Would add benefit to the test if we knew what the values were and how the adjustment amount was derived.